src/main/java/org/alliancegenome/curation_api/base/BaseCrudRESTInterface.java

```diff
@@ -37,7 +37,7 @@
 
     @POST
     @Path("/find")
-    @JsonView(View.FieldsOnly.class)
+    @JsonView(View.FieldsAndLists.class)
```

Comment:
This may be problematic in the future. For objects that references back and forth with each other.

Reply:
It would be better to override this method in the interface and change the JsonView there.